pkg/cluster/builder.go

```diff
@@ -154,6 +154,28 @@ func (b FileSpecBuilder) getBundles(manifestReader *manifests.Reader) (*releasev
 	return bundles.Read(b.reader, bundlesURL)
 }
 
+func (b FileSpecBuilder) getEksaRelease(mReader *manifests.Reader) (*releasev1.EksARelease, error) {
+	if b.bundlesManifestURL == "" {
```

Comment:
Will this be `""` in airgap environments?

Reply:
"" is there if bundles-override is NOT set. You can see a similar check happening [here](https://github.com/aws/eks-anywhere/pull/6299/files/7aa3d4094a048ce0eeaf5b7c03c0d9a5bc002f9e#diff-5a573820e4737e0df734c87f7ff19388a4717953c43f7406572b012a52451338R150)